src/mpid/ch4/netmod/include/netmod.h

```diff
@@ -51,6 +51,9 @@ typedef int (*MPIDI_NM_am_isend_reply_t) (MPIR_Context_id_t context_id, int src_
 typedef size_t(*MPIDI_NM_am_hdr_max_sz_t) (void);
 typedef size_t(*MPIDI_NM_am_eager_limit_t) (void);
 typedef size_t(*MPIDI_NM_am_eager_buf_limit_t) (void);
+typedef bool(*MPIDI_NM_am_check_eager_limit_t) (MPI_Aint am_hdr_sz, MPI_Aint data_sz,
+                                                const void *data, MPI_Count count,
+                                                MPI_Datatype datatype, MPIR_Request * sreq);
```

Comment:
The name seems misleading since it is no longer a simple check of "eager limit". How about just `MPIDI_NM_am_check_eager`? Feel free to suggest anything better.

Reply:
Good point.